app/js/sanitiseAdmin.js

```diff
@@ -0,0 +1,23 @@
+
+
+//add in isEmailValid from validateUserAdditions to validate edit email
+
+//to make sure name and email fields arent empty
+function isEmpty (field){
+    if (field.value === "") {
+        field.style.background = "red"
```

Comment:
This doesn't seem to work. I was expecting the boxes to change colour to red? Perhaps the issue is that the background could be "background-color" ??